TSC2/README.md

```diff
@@ -0,0 +1,114 @@
+# ODK TSC Landing Page
+
+The Technical Steering Committee is the technical governing body for Open Data Kit 2 core projects.
+
+## Quick Links
+
+- [Meeting Agendas and Minutes](https://docs.google.com/document/d/1e7oayqi0eTHVJx_q9-S-RgPYyffnqak3nnQq2Dps3Pk/edit)
+- [ODK Roadmap](https://github.com/opendatakit/roadmap/projects/2)
+- [Uberconference Room](https://www.uberconference.com/opendatakit)
+- [TSC Forum Category](https://forum.opendatakit.org/c/governance/tsc)
+- [TSC Forum Tag](https://forum.opendatakit.org/tags/tsc)
+- ODK 2 Project Pages: [Tables](https://github.com/opendatakit/tables), [Survey](https://github.com/opendatakit/survey), [Services](https://github.com/opendatakit/services), [Documentation](https://github.com/opendatakit/docs), [Application Designer](https://github.com/opendatakit/app-designer), [Suitcase](https://github.com/opendatakit/suitcase), [Sync-Endpoint](https://github.com/opendatakit/sync-endpoint)
+- [TSC Governance Guidelines](https://github.com/opendatakit/governance/blob/master/TECHNICAL-STEERING-COMMITTEE.md)
+- [Public Governance Page with TSC Members & Photos](https://opendatakit.org/community/governance/)
+- [Meeting Facilitator Checklist](#meeting-facilitator-checklist)
+
+For more details on the TSC's responsibilities, see the [TSC governance](https://github.com/lognaturel/governance/blob/master/TECHNICAL-STEERING-COMMITTEE.md) approved on Nov 7, 2017.
```

Comment:
I think we should update this note to distinguish that this link is to TSC 1, and that we are using their same responsibilities for now.

The "approved on" date should probably be updated too.